src/main/java/Task.java

```diff
@@ -0,0 +1,32 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
```

Comment:
Avoid complicated expressions, consider seperating the logic to make the code more understandable. Also remember to remove uneeded comments after making the statements clear (i.e. comments should avoid stating the obvious).